nextflow.config

```diff
@@ -190,6 +192,7 @@ profiles {
         docker.userEmulation   = { params.use_gatk_spark ? false : true }.call()
         podman.enabled         = false
         shifter.enabled        = false
+        docker.runOptions      = '-u $(id -u):$(id -g)'
```

Comment:
I am getting anxious about any spark related issues :D

Reply:
a bit too, we'd need to try this out properly, I'll make an issue

Reply:
https://github.com/nf-core/sarek/issues/1362